packages/yourloops/pages/login/login.tsx

```diff
@@ -130,13 +188,24 @@ function Login(): JSX.Element {
       log.debug("Logged user:", user);
       // The redirect is done by packages/yourloops/components/routes.tsx#PublicRoute
     } catch (reason: unknown) {
+      let action: JSX.Element | null = null;
       let errorMessage = errorTextFromException(reason);
       if (errorMessage === "error-account-lock") {
         errorMessage = t(errorMessage, { delayBeforeNextLoginAttempt: appConfig.DELAY_BEFORE_NEXT_LOGIN_ATTEMPT });
+      } else if (errorMessage === "email-not-verified") {
+        action = (
+          <ButtonResendActivationLink
+            username={username}
+            log={log}
+            setResendActivationLinkInProgress={setResendActivationLinkInProgress}
+          />
+        );
+        errorMessage = t(errorMessage);
       } else {
         errorMessage = t(errorMessage);
       }
-      alert.error(errorMessage);
+
```

Comment:
the else is not necessary :

      errorMessage = t(errorMessage);
      alert.error(errorMessage, action);

Reply:
Sadly it is.
If not the error messages others than "error-account-lock" and "email-not-verified" won't be translated.
But yes, it's a little weird, I will change this part.